src/main/java/seedu/heymatez/logic/parser/ParserUtil.java

```diff
@@ -40,6 +42,8 @@
 
     public static final String MESSAGE_INVALID_INDEX = "Index is not a non-zero unsigned integer.";
 
+    public static final String MESSAGE_NON_POSITIVE_INDEX = "Index is a non positive integer.";
```

Comment:
I think there is a typo here, should have a hyphen 'non-positive'

Reply:
i actually did that before but there was apparently a checkstyle error